server_test.py

```diff
@@ -0,0 +1,109 @@
+""" Service requests for accessible objects """
```

Comment:
Yep it's the old flask implementation but may be nice to leave for debugging purposes (prints out really nice JSON errors)